.github/workflows/ci-dgraph-upgrade-tests.yml

```diff
@@ -0,0 +1,52 @@
+name: ci-dgraph-upgrade-tests
```

Comment:
ci-dgraph-upgrade-and-integration-tests

Reply:
I'm not sure this is good terminology since our ci-dgraph-tests and ci-dgraph-load-tests are also actually integration tests, @skrdgraph  you have thoughts on this?